src/adapter/mysql.cr

```diff
@@ -43,7 +43,7 @@ class Granite::Adapter::Mysql < Granite::Adapter::Base
     end
   end
 
-  def insert(table_name, fields, params)
+  def insert(table_name, primary_name, fields, params)
```

Comment:
Is it possible to get mysql and sqlite to generate a UUID as well?

Reply:
no experience on mysql and sqlite,  rails use before_create to generate UUID, we generate in insert transaction automatically?